src/main/java/com/softserve/academy/spaced/repetition/utils/exceptions/TransactionException.java

```diff
@@ -0,0 +1,11 @@
+package com.softserve.academy.spaced.repetition.utils.exceptions;
+
+public class TransactionException extends ApplicationException {
```

Comment:
Please rename to PointsTransactionException, because now it looks like something related to database transaction.